### ISSUE TITLE
Full support for draft/master branch

### ISSUE THREAD

**Issue by Bellfalasch** [enonic/cms2xp#35]
Migrated data suffers from one pretty big issue - that we only get "draft"-data. We cannot possibly see what was online or not in CMS, meaning we more or less manually need to publish individual items in XP based on comparison with CMS. Or, we can publish everything and keep our fingers crossed. Both alternatives are really bad.

We need a way to get this data with us from the migration. Either by solving it like #28 and then script the publishing in XP - or by migrating two branches. This could possibly also solve #33 that reports lost drafts. However, two branches will make the export big, so that would require an option/toggle.

If this is more likely to be easier done in pure XP, I suggest focusing on #28 and in addition writing a publishing-script that uses these values to move things from draft to master.

**Comment by sigdestad** [comment]
How about we only export content that is online (possibly including pending and exipred items). That way you can safely publish everything?

It could be a flag if user wants to include drafts (of unpublished items), but then user must deal with it himself?

**Comment by Bellfalasch** [comment]
@sigdestad Yeah that would work. We might have some cases with "but where's that Christmas-article we usually re-publish every year" but at least better than than publishing-problems we have now. I agree drafts of unpublished items are perhaps not needed.

But what do you think about #28, I think that + an XP script could handle publishing too. Then we don't create two branches, but at least "tag" all contents with their CMS status and can act upon it with scripts.
